network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/image_grid_block.html

```diff
@@ -1,9 +1,10 @@
 {% load wagtailcore_tags wagtailimages_tags %}
-
-<div class="row my-default">
-{% for block in self.grid_items %}
-	{% with value=block count=self.grid_items|length %}
-	{% include "./figure_block.html" %}
-	{% endwith %}
-{% endfor %}
```

Comment:
Opening its own container and row, then following the rest of the original template